packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application, Server, api,
+  OpenApiSpec, ParameterObject,
+  ServerRequest, ServerResponse, parseOperationArgs, writeResultToResponse,
+} from '../../../src';
+import {ParsedRequest, OperationArgs, FindRoute, InvokeMethod} from '../../../src/internal-types';
+import {expect, Client, createClientForServer} from '@loopback/testlab';
+import {givenOpenApiSpec} from '@loopback/openapi-spec-builder';
+import {inject, Constructor, Context} from '@loopback/context';
+
+/* # Feature: Routing
+ * - In order to build REST APIs
+ * - As an app developer
+ * - I want the framework to handle the request to controller method routing
+ * - So that I can focus on my implementing the methods and not the routing
+ */
+describe('Routing', () => {
```

Comment:
@superkhau this is exactly how existing tests in routing.acceptance.ts are implemented.